### ISSUE TITLE
Validation is failing with message "No evaluator available" when trying to run sample code

### ISSUE THREAD

**Issue by tunnelWithAC** [telus-oss/protovalidate-net#20]
Hi,

I've been trying to use this package in a test project but I am facing issues when validating an object.

I have an example [console app](https://github.com/tunnelWithAC/dotnet-proto-validate/tree/master/src/ConsoleExample) in this folder and here is the [Program.cs](https://github.com/tunnelWithAC/dotnet-proto-validate/blob/master/src/ConsoleExample/Program.cs).

This is the command I used to generate the proto artifacts `protoc -I=. -I ./protovalidate/proto/protovalidate --csharp_out=.. ./user.proto --descriptor_set_out=../Protoset/user.protoset` command

Any help would be appreciated. Thanks

**Comment by aneshas** [comment]
@tunnelWithAC - same here. Have you been able to resolve this?

**Comment by tunnelWithAC** [comment]
@aneshas yes, I have an example [here](https://github.com/tunnelWithAC/protovalidate-experimentation/blob/master/src/dotnet/ConsoleValidator/Validators/ProtoValidator.cs).

Basically just set `options.PreLoadDescriptors` to `false`

**Comment by aneshas** [comment]
Man - I owe you a coffee. Spent the whole day yesterday fighting this. I tried setting the `PreLoadDescriptors` to false and dropped it right after that - but now I tried it again, still didn't work until you also set `DisableLazy` to false. In any case, you are a lifesaver thanks. @tunnelWithAC
